kuzzle/query.go

```diff
@@ -47,7 +45,7 @@ func (k *Kuzzle) Query(query *types.KuzzleRequest, options types.QueryOptions, r
 		query.Volatile = options.Volatile()
 
 		mapped := make(map[string]interface{})
-		json.Unmarshal(query.Volatile, &mapped)
+		_ = json.Unmarshal(query.Volatile, &mapped)
```

Comment:
What the point of using assignment operator if we don't use the variable after ?

Reply:
This function doesn't return an error, so we can't assign it to the variable. But if you don't do this, linter shell warn you